vendor/src/github.com/docker/notary/Makefile

```diff
@@ -92,14 +92,21 @@ build: go_version
 	@echo "+ $@"
 	@go build -tags "${NOTARY_BUILDTAGS}" -v ${GO_LDFLAGS} ./...
 
+# When running `go test ./...`, it runs all the suites in parallel, which causes
```

Comment:
It only removes unused packages and files that we can _definitely_ say are
completely unused by our code like "*_test.go", so this existing is correct
(I don't think there's a definitive way our script could detect whether
this is used or not -- we already hard-code one directory that only
contains "*.c" files that we need and the script wants to remove otherwise,
for example).